src/test/java/com/kalessil/phpStorm/phpInspectionsEA/lang/UnnecessaryElseInspectorTest.java

```diff
@@ -0,0 +1,16 @@
+package com.kalessil.phpStorm.phpInspectionsEA.lang;
+
+import com.intellij.testFramework.fixtures.CodeInsightFixtureTestCase;
+import com.kalessil.phpStorm.phpInspectionsEA.inspectors.languageConstructions.UnnecessaryElseInspector;
+
+/**
+ * (c) Funivan <alotofall@gmail.com>
+ */
+public class UnnecessaryElseInspectorTest extends CodeInsightFixtureTestCase {
```

Comment:
`final` + I dropping all class comments in tests normally. There is git history to track an author.